triton-vm/src/table/ram_table.rs

```diff
@@ -532,16 +533,17 @@ pub struct RamTableChallenges {
 }
 
 impl TableChallenges for RamTableChallenges {
-    type Id = RamTableChallengesId;
+    type Id = RamTableChallengeId;
 
+    #[inline]
     fn get_challenge(&self, id: Self::Id) -> XFieldElement {
         match id {
-            RamTableChallengesId::BezoutRelationIndeterminate => self.bezout_relation_indeterminate,
-            RamTableChallengesId::ProcessorPermIndeterminate => self.processor_perm_indeterminate,
-            RamTableChallengesId::ClkWeight => self.clk_weight,
-            RamTableChallengesId::RamvWeight => self.ramv_weight,
-            RamTableChallengesId::RampWeight => self.ramp_weight,
-            RamTableChallengesId::AllClockJumpDifferencesMultiPermIndeterminate => {
+            RamTableChallengeId::BezoutRelationIndeterminate => self.bezout_relation_indeterminate,
+            RamTableChallengeId::ProcessorPermIndeterminate => self.processor_perm_indeterminate,
+            RamTableChallengeId::ClkWeight => self.clk_weight,
+            RamTableChallengeId::RamvWeight => self.ramv_weight,
+            RamTableChallengeId::RampWeight => self.ramp_weight,
+            RamTableChallengeId::AllClockJumpDifferencesMultiPermIndeterminate => {
```

Comment:
Let's use a `use` here.
```suggestion
			use RamTableChallengeId::*; // might have to live one scope above
            BezoutRelationIndeterminate => self.bezout_relation_indeterminate,
            ProcessorPermIndeterminate => self.processor_perm_indeterminate,
            ClkWeight => self.clk_weight,
            RamvWeight => self.ramv_weight,
            RampWeight => self.ramp_weight,
            AllClockJumpDifferencesMultiPermIndeterminate => {
```